hw/ip/gpio/dv/cov/gpio_cov_excl.el

```diff
@@ -1,34 +1,63 @@
 // Copyright lowRISC contributors (OpenTitan project).
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
+
 //==================================================
 // This file contains the Excluded objects
-// Generated By User: gaurangg
+// Generated By User: marcelo
 // Format Version: 2
-// Date: Wed Oct 30 17:22:12 2019
+// Date: Fri Feb 21 12:40:56 2025
 // ExclMode: default
 //==================================================
-CHECKSUM: "3109872245 3727953184"
-INSTANCE: tb.dut.u_reg.u_data_in
-ANNOTATION: "[UNR]"
-Block 1 "0" "assign wr_en = de;"
-CHECKSUM: "1352181506 3803838024"
+CHECKSUM: "1706182284 132761700"
+INSTANCE: tb.dut.u_reg.u_reg_if.u_rsp_intg_gen
+ANNOTATION: "The signal tl_i.d_user is tied to zero in the tlul_adapter_reg.sv which comes from tl_o_pre. Because of that, the rsp_intg it will be constant and can be excluded from the coverage analysis."
+Block 1 "461445014" "assign rsp_intg = tl_i.d_user.rsp_intg;"
+ANNOTATION: "The signal tl_i.d_user is tied to zero in the tlul_adapter_reg.sv which comes from tl_o_pre. Because of that, the rsp_intg it will be constant and can be excluded from the coverage analysis."
+Block 2 "2643129081" "assign data_intg = tl_i.d_user.data_intg;"
+CHECKSUM: "2099741489 2073313596"
+INSTANCE: tb.dut.u_reg.u_data_in.wr_en_data_arb
+ANNOTATION: "The line 41 where `wr_en = de` is constant because the \"de\"  is wired to 1 in gpio.sv (line 104).  `hw2reg.data_in.de = 1'b1;`
+So it will not be hit by the coverage."
+Block 1 "494386457" "assign wr_en = de;"
+CHECKSUM: "295805079 2786204945"
 INSTANCE: tb.dut.u_reg.u_masked_out_lower_mask
-ANNOTATION: "[UNR]"
-Block 1 "0" "assign qs = d;"
-CHECKSUM: "1352181506 3803838024"
-INSTANCE: tb.dut.u_reg.u_masked_out_upper_mask
-ANNOTATION: "[UNR]"
-Block 1 "0" "assign qs = d;"
-CHECKSUM: "1352181506 3803838024"
+ANNOTATION: "The hardware always signals d=0 (through hw2reg.masked_out_lower.mask.d) so this continuous assignment will not execute."
+Block 2 "1004401251" "assign qs = d;"
+ANNOTATION: "The hardware always signals d=0 (through hw2reg.masked_out_lower.mask.d) so this continuous assignment will not execute."
+Block 1 "531057943" "assign ds = d;"
+CHECKSUM: "295805079 2786204945"
 INSTANCE: tb.dut.u_reg.u_masked_oe_lower_mask
-ANNOTATION: "[UNR]"
-Block 1 "0" "assign qs = d;"
-CHECKSUM: "1352181506 3803838024"
+ANNOTATION: "The hardware always signals d=0 (through hw2reg.masked_out_lower.mask.d). So this is continuous assignment will not execute."
```

Comment:
Being a little more specific (than I was [here](https://github.com/lowRISC/opentitan/pull/25767#discussion_r1969544036)), I think this annotation talks about `masked_out_lower` but it applies to `masked_ao_lower_mask`.

I think that's true for the other `oe` lines below as well.